docker-compose.link.yaml

```diff
@@ -0,0 +1,5 @@
+version: '3.8'
+networks:
+  default:
+    external:
+      name: ffc-pay
```

Comment:
Required so containers are available across docker compose projects.

Same approach taken in payment service, SFI and demo etc